src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1103,6 +1103,12 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
         $translator = $container->findDefinition('translator.default');
         $translator->addMethodCall('setFallbackLocales', [$config['fallbacks']]);
 
+        if (\array_key_exists('cache_dir', $config)) {
+            $defaultOptions = $translator->getArgument(4);
+            $defaultOptions += ['cache_dir' => $config['cache_dir']];
```

Comment:
`cache_dir` is already defined so that `+=` will be a no-op, isn't it?

Reply:
tanks for your feedback. Fixed